test/OpenTelemetry.Instrumentation.AspNetCore.Tests/BasicTests.cs

```diff
@@ -61,17 +61,9 @@ public async Task SuccessfulTemplateControllerCallGeneratesASpan()
 
             void ConfigureTestServices(IServiceCollection services)
             {
-                var openTelemetry = OpenTelemetrySdk.Default.EnableOpenTelemetry(
+                var openTelemetry = OpenTelemetrySdk.EnableOpenTelemetry(
                 (builder) => builder.AddRequestInstrumentation()
                 .SetProcessorPipeline(p => p.AddProcessor(n => spanProcessor.Object)));
-
-                /*
-                services.AddSingleton<TracerFactory>(_ =>
```

Comment:
See how the old code was doing `services.AddSingleton<TracerFactory>`? There's a handful of spots like this in the test code. In the version I did in our "sandbox" earlier I preserved that for the new style (`services.AddSingleton<OpenTelemetrySdk>(_ =>`) because I think what the test is trying to do is make sure the dependency injection works as expected. The tests are a bit redundant but you might want to update them for correctness?